docs/getting-started.md

```diff
@@ -23,7 +23,7 @@ Remember to call `@react-native-community/cli init` from the place you want your
 <!-- See https://www.npmjs.com/package/react-native?activeTab=versions for the RN version tags. -->
 
 ```bat
-npx --yes @react-native-community/cli@next init <projectName> --version "nightly"
+npx --yes @react-native-community/cli@next init <projectName> --version "latest"

```

Comment:
This main page in the docs needs to stay the same as it reflects how to build against main/canary/the perpetual next version.

```suggestion
npx --yes @react-native-community/cli@next init <projectName> --version "nightly"

```